luxonis_train/attached_modules/losses/adaptive_detection_loss.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         reduction: Literal["sum", "mean"] = "mean",
         class_loss_weight: float = 1.0,
         iou_loss_weight: float = 2.5,
+        per_class_weights: Optional[List[float]] = None,
```

Comment:
Luxonis-train is py3.10 so type annotation can be:
```suggestion
        per_class_weights: list[float] | None = None,
```